routes/api.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+use Illuminate\Support\Facades\Route;
+use App\Http\Controllers\CreateCityController;
+use App\Http\Controllers\DestroyCityController;
+use App\Http\Controllers\GetAllCitiesController;
+use App\Http\Controllers\UpdateCityController;
+
+Route::get('/cities', GetAllCitiesController::class);
+Route::post('/cities', CreateCityController::class);
+Route::patch('/cities/{id}', UpdateCityController::class);
+Route::delete('/cities/{id}', DestroyCityController::class);
```

Comment:
you could add a name for each route and then use that name instead of hardcoding the url in your views 
```suggestion
Route::get('/cities', GetAllCitiesController::class)->name('cities');
Route::post('/cities', CreateCityController::class)->name('cities.store');
Route::patch('/cities/{id}', UpdateCityController::class)->name('cities.update');
Route::delete('/cities/{id}', DestroyCityController::class)->name('cities.destroy');
```

Reply:
also to avoid repetition of /cities you could use a prefix and group all your routes by the same prefix. Check it here: https://laravel.com/docs/11.x/routing#route-group-prefixes

This will be also useful when working with middleware and authorization.